apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -377,7 +377,15 @@ export const eventsToLink = async (
   const event = ctx.session.selectedEvent;
   if (event) {
     range
-      .text(`${event.isLinkedToCurrentChat ? "✅" : ""} ${event.eventName}`)
+      .text(
+        `↰ ${event.isLinkedToCurrentChat ? "✅" : ""} ${event.eventName}`,
+        async (ctx) => {
+          if (!(await senderIsAdmin(ctx))) return;
```

Comment:
do we need to check for Admin in a DM?

Reply:
Yes, this prevents non admins from clicking the buttons. This is the selectedEvent in the `/manage` command btw